### PR TITLE
Pipe: Add a configuration parameter to control the number of rows of the tablet used within the pipe module

### PR DESCRIPTION
**Pull request by DanielWang2035**
Currently, tablets in pipe transfer are created with default rowSize. If data fransfer load is high, there  will be too many thrift requests of tablet transfer. So we can add a knob to specify rowSize of pipe tablets.

### CODE REVIEW

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 1]
```diff
@@ -972,6 +972,9 @@ cluster_name=defaultCluster
 # Noted that: this should be less than or equals to realtimeExtractorPendingQueueCapacity
 # pipe_extractor_pending_queue_tablet_limit=64
 
+# The row size of tablets created in the extractor stage.
+# pipe_extractor_tablet_row_size=65536
```
Comment (SteveYurongSu) [inline iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties:976]:
```suggestion
# pipe_datastructure_tablet_row_size=65535
```

**Review by SteveYurongSu — CHANGES_REQUESTED** [review]
Only a minor issue about naming

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties (hunk 1 above)
Comment (DanielWang2035) [reply iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties]: changed

**Review by SteveYurongSu — APPROVED** [review]

**Comment by SteveYurongSu** [comment]
Please cp this commit to rel/1.2, thanks!

**Pull request merged**